packages/sanity/src/core/bundles/components/dialog/BundleDetailsDialog.tsx

```diff
@@ -14,14 +16,16 @@ interface BundleDetailsDialogProps {
   onCancel: () => void
   onSubmit: () => void
   bundle?: BundleDocument
+  origin?: 'structure' | 'release-plugin'
```

Comment:
COULD: export `OriginInfo` from `releases.telemetry` and use that here instead

Reply:
Great suggestion, done :)